src/sentry/api/endpoints/organization_transaction_anomaly_detection.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime
+
+from django.conf import settings
+from rest_framework.request import Request
+from rest_framework.response import Response
+from urllib3 import Retry, connection_from_url
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.api.utils import get_date_range_from_params
+from sentry.snuba.discover import timeseries_query
+from sentry.utils import json
+
+
+def get_anomalies(snuba_io):
+    ads_connection_pool = connection_from_url(
+        settings.ANOMALY_DETECTION_URL,
+        retries=Retry(
+            total=5,
+            status_forcelist=[408, 429, 502, 503, 504],
+        ),
+        timeout=settings.SENTRY_SNUBA_TIMEOUT,
```

Comment:
Should we be explicit and have a timeout specifically for the anomaly service? Otherwise there may be odd side effects if this snuba timeout ever needs to change